climkern/tests/test_frontend.py

```diff
@@ -1,14 +1,23 @@
-from climkern.frontend import *
 import xarray as xr
 
-ctrl, pert = tutorial_data('ctrl'),tutorial_data('pert')
+from climkern.frontend import *
+
+ctrl, pert = tutorial_data("ctrl"), tutorial_data("pert")
```

Comment:
Consider adding definitions (here or in the documentation) about any different tutorial data.

Reply:
You're right, I'll add some documentation to it. Thanks!